xcell/mappers/mapper_WIxSC.py

```diff
@@ -123,8 +141,8 @@ def get_nz(self, dz=0, return_jk_error=False):
                 z, nz, nz_jk = get_DIR_Nz(c_s, c_p,
                                           ['W1c', 'W2c', 'Bcc', 'Rcc'],
                                           zflag='Zspec',
-                                          zrange=[0, 1.],
-                                          nz=100,
+                                          zrange=[0, 0.6],
+                                          nz=150,
```

Comment:
Why 150 instead of 100?

Reply:
Just keeping the redshift spacing the same as 2MPZ (100 points from 0 to 0.4)